eng/pipelines/libraries/build-job.yml

```diff
@@ -141,6 +141,11 @@ jobs:
             - template: /eng/pipelines/common/upload-intermediate-artifacts-step.yml
               parameters:
                 name: Libraries_AllConfigurations
+                publishPackagesCondition: >-
+                  or(
+                     eq(variables['_librariesBuildProducedPackages'], true),
+                     eq(variables['Build.SourceBranchName'], 'main'),
+                     eq(variables['System.PullRequest.TargetBranch'], 'main')) 
```

Comment:
@safern what is this about?

Reply:
This is because we expect the packages to be se to true on main always. So if it is set to false we want this step to run and fail, so that we don't silently stop producing packages and miss it on a PR build.

Reply:
Ah I missed the `or` condition. I read it as an `and` which made me believe this won't run in servicing. Thanks for clarifying.